dist/beans/aggregators/aggregators.ts

```diff
@@ -10,37 +10,54 @@ import {ScaleAggregator} from "./scale_aggregator";
 import {SmaAggregator} from "./sma_aggregator";
 import {TrimAggregator} from "./trim_aggregator";
 
-export const AGGREGATORS = [
+const RANGE_AGGREGATORS = [
     new RangeAggregator("avg"),
-    new RangeAggregator("dev"),
     new RangeAggregator("count"),
+    new RangeAggregator("dev"),
     new RangeAggregator("first"),
     new RangeAggregator("gaps"),
     new RangeAggregator("last"),
     new RangeAggregator("least_squares"),
     new RangeAggregator("max"),
+    // merge intentionally excluded; mportal adds it automatically now
     new RangeAggregator("min"),
-    new RangeAggregator("merge"),
     new RangeAggregator("movingWindow"),
-    new PercentileAggregator(),
-    new ApdexAggregator(),
-    new SmaAggregator(),
     new RangeAggregator("sum"),
+];
+
+export const AGGREGATORS = RANGE_AGGREGATORS.concat([
```

Comment:
If an aggregator is not in the `RANGE_AGGREGATORS` list does that mean it is not considered to return a scalar? Or perhaps you could elaborate on the what these two categorizations mean (via a comment)

Reply:
They're unrelated sets. I didn't think entirely reworking the Aggregator datamodel is a change worth making here. 

We could register each of these properties as a boolean (scalar, range) for every Aggregator, but as-is, some of these aggregators are little more than a name string.

I think this change is one of the ones that has not yet been merged (and isn't part of my change).

Reply:
Resolving as I've removed these unrelated changes.